scripts/pull_images.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Use the CONTAINER_RUNTIME from the environment, or default to docker if it's not set.
+CONTAINER_RUNTIME="${CONTAINER_RUNTIME:-docker}"
+echo "Using container tool: ${CONTAINER_RUNTIME}"
+
+# Set a default EPP_TAG if not provided
+export EPP_TAG="${EPP_TAG:-latest}"
+# Set a default VLLM_SIMULATOR_TAG if not provided
+export VLLM_SIMULATOR_TAG="${VLLM_SIMULATOR_TAG:-v0.5.0}"
+# Set the default routing side car image tag
+export SIDECAR_TAG="${SIDECAR_TAG:-dev}"
+
+EPP_IMAGE="ghcr.io/llm-d/llm-d-inference-scheduler:${EPP_TAG}"
+VLLM_SIMULATOR_IMAGE="ghcr.io/llm-d/llm-d-inference-sim:${VLLM_SIMULATOR_TAG}"
+ROUTING_SIDECAR_IMAGE="ghcr.io/llm-d/llm-d-routing-sidecar:${SIDECAR_TAG}"
+
+TARGETOS="${TARGETOS:-$(go env GOOS)}"
```

Comment:
This will not work when running on a Mac.

```suggestion
TARGETOS="${TARGETOS:-linux}"
```